reference/v8.json

```diff
@@ -1861,6 +1904,9 @@
       "sdk-support": {
         "basic functionality": {
           "js": "0.27.0"
+        },
+        "data-driven styling": {
+          "js": "Not supported"
```

Comment:
I'm seeing this being treated like a version number locally:
<img width="616" alt="screen shot 2017-01-26 at 7 09 21 pm" src="https://cloud.githubusercontent.com/assets/4974151/22355901/8127d14a-e3fb-11e6-8db9-0dacd8281383.png">

I think just removing the entry fixes it -- @jfirebaugh is there some other reason we need it in here?  If so we can tweak the html template to handle this...